debian/control

```diff
@@ -14,10 +14,10 @@ Vcs-Git: https://github.com/AtlasOfLivingAustralia/ala-sensitive-data-service.gi
 
 Package: ala-sensitive-data-service
 Architecture: all
-Depends: ${misc:Depends}, openjdk-8-jdk, ca-certificates, tar (>= 1.0), rsync, 
- curl, adduser
+Depends: ${misc:Depends}, openjdk-8-jdk, ca-certificates, tar (>= 1.0), curl,
+ ala-namematching-service
```

Comment:
After talking with @charvolant and see this dep here (I thought that was missing) all LGTM.